experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -384,13 +385,15 @@ export class IdCompressor {
 	public constructor(
 		public readonly localSessionId: SessionId,
 		public readonly reservedIdCount: number,
-		attributionId?: AttributionId
+		attributionId?: AttributionId,
+		private readonly logger?: ITelemetryLogger
 	) {
 		assert(reservedIdCount >= 0, 'reservedIdCount must be non-negative');
 		if (attributionId !== undefined) {
 			assertIsUuidString(attributionId);
 		}
 		this.localSession = this.createSession(localSessionId, attributionId);
+		this.logger = logger;
```

Comment:
You don't need this assignment anymore :)

Reply:
😮